### DISCUSSION TITLE
Modbus communication in Disabled mode

### DISCUSSION THREAD

**Discussion by edrocket** [SmartEVSE/SmartEVSE-2#53] [Q&A]
HI, I like to control the charging current and/or disable charging (to working with dynamic pricing).
I can only communicatie with Modbus when Load Balancing is on (Master).
The I can set charging current between 6A and 16A (function A6) or disable charging (using function A7, enable/disable access).

However, I like to communicatie when the device is not in Load Balancing mode, because al the messages to "slave nodes" disturb my normal modbus communication (I have other devices on the RS485 but).

I scanned the source code, but at a first glance, modbus communication should be possible (evse as slave) when LoadBL = 0.
However, no communication at all when Load Balancing is disabled, only in Master.

Any thoughts?

Ps. Is there a better way to stop charging other than changing the Acces (0xa7) state?

**Comment by edrocket** [comment]
Ok, I was looking at the wrong branch, its fixed in 2.30...

Still maybe someone to answer my remaining question for what is the suggested way to delay charging, besides using the Access register

**Comment by mstegen** [comment]
There is no easy way. The best way is set the controller to Node 1, and mimic a controller configured as Master.

Then if a charge request comes in from Node1, you can just ignore it, until you allow it to start charging.
The Node controller requires a status readout, and update of chargecurrent once every 2 seconds.

**Comment by edrocket** [comment]
Thanks for the feedback. I understand. Then I will still "use" the Access function, because that works more or less the same in case I want to delay a full charge. 
Or maybe 0x0002 Charging current (0: no current), which wasn't available in the 2.20 version (writing 0 to Charging Current revealed in 6A charging)
